### PR TITLE
Candidate and admin login changes

### PR DESCRIPTION
**Pull request by Shreya-Kanani**

### CODE REVIEW

File: src/app/core/auth/components/login/login.component.html [hunk 1]
```diff
@@ -35,11 +35,20 @@ <h2 class="title">Welcome</h2>
               <mat-checkbox color="primary" class="font14"
                 >Remeber me</mat-checkbox
               >
+              <a href="#" class="forgot-password" *ngIf="!isAdmin"
```
Comment (ShubhamPadiyaTatvasoft) [inline src/app/core/auth/components/login/login.component.html:38]: need of href?

File: src/app/core/auth/components/login/login.component.html [hunk 2]
```diff
@@ -35,11 +35,20 @@ <h2 class="title">Welcome</h2>
...
+              >
             </div>
           </div>
           <div class="col-12 animation-target">
             <button mat-flat-button color="primary" class="w-100">Login</button>
           </div>
+          <div class="col-12 animation-target mt-5" *ngIf="!isAdmin">
+            <p class="account">
+              Don't have an account?
+              <a href="#" class="register">Register Now!</a>
```
Comment (ShubhamPadiyaTatvasoft) [inline src/app/core/auth/components/login/login.component.html:49]: there is no logic for register page redirection

File: src/app/core/auth/components/login/login.component.scss [hunk 3]
```diff
@@ -74,6 +74,25 @@
...
+
+.register {
+  color: rgba(72, 117, 183, 255);
+  font-size: 15px;
+  font-weight: 700;
+  text-decoration: none;
+}
+.account {
+  font-size: 14px;
+  // font-weight: 600;
```
Comment (ShubhamPadiyaTatvasoft) [inline src/app/core/auth/components/login/login.component.scss:92]: remove  comment

File: src/app/core/auth/auth-routing.module.ts [hunk 4]
```diff
@@ -10,6 +10,10 @@ const routes: Routes = [
     path: '',
     component: LoginComponent,
   },
+  {
+    path: 'admin/login',
```
Comment (ShubhamPadiyaTatvasoft) [inline src/app/core/auth/auth-routing.module.ts:14]: use enum

File: src/app/core/auth/components/login/login.component.html (hunk 2 above)
Comment (Shreya-Kanani) [reply src/app/core/auth/components/login/login.component.html:49]: That was done by rajvi ma'am

**Review by ShubhamPadiyaTatvasoft — APPROVED** [review]

**Review by UrvishaNayee — APPROVED** [review]

**Pull request merged**